src/Symfony/Bridge/Twig/CHANGELOG.md

```diff
@@ -2,14 +2,11 @@ CHANGELOG
 =========
 
 5.3
------
+---
 
 * Add a new `markAsPublic` method on `NotificationEmail` to change the `importance` context option to null after creation
-
-5.3.0
------
-
-* Added a new `serialize` filter to serialize objects using the Serializer component
+* Add a new `serialize` filter to serialize objects using the Serializer component
```

Comment:
I think you need to keep `5.3.0` changes as is, and add your new line to `5.3`

Reply:
Yep I rollback to how it was!